qiskit_experiments/base_experiment.py

```diff
@@ -288,3 +289,7 @@ def set_analysis_options(self, **fields):
             fields: The fields to update the options
         """
         self._analysis_options.update_options(**fields)
+
+    def _postprocess_transpiled_circuits(self, circuits: List[QuantumCircuit]):
```

Comment:
My goal here is to minimize as much as possible taking over processes which the base class should handle. I don't need control over the transpilation stage itself; only a peek into its results (similar to the post processing in use in the analysis class). Of course, I'm happy to hear any suggestion you or @chriseclectic might have.

Reply:
I think this sort of step could be useful in general. We might want to add backend and run_options to it since they could be useful (eg for adding measurement level metadata for a calibration experiment) eg

```python
def `_postprocess_transpiled_circuits(self, circuits, backend=None, **run_options):
    """Additional post-processing of transpiled circuits before running on backend"""    
    pass
```